### PR TITLE
config: Add support for font slant and weight

### PR DESCRIPTION
**Pull request by jlindgren90**
Adds support for bold and italic fonts, following the Openbox syntax.

### CODE REVIEW

File: include/common/font.h [hunk 1]
```diff
@@ -4,11 +4,25 @@
...
+};
+
 struct font {
 	char *name;
 	int size;
+	enum font_slant slant;
+	enum font_weight weight;
 };
 
+struct _PangoFontDescription *font_to_pango_desc(struct font *font);
```
Comment (johanmalm) [inline include/common/font.h:24]: Is the use of struct here to avoid being pango in the includes?

**Comment by johanmalm** [comment]
LGTM

File: include/common/font.h (hunk 1 above)
Comment (jlindgren90) [reply include/common/font.h:24]: Yes, especially now that `font.h` is pulled in by `rcxml.h` as well

Comment (johanmalm) [reply include/common/font.h:24]: Nice. It took me a few seconds before the penny dropped. Like it.

**Comment by johanmalm** [comment]
I have update the scope document.
https://github.com/labwc/labwc-scope/commits/main

**Pull request merged**